.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+# D203: 1 blank line required before class docstring
+# F401: 'identifier' imported but unused
+# E402: module level import not at top of file
+# C901: too complex
+exclude = venv,venv3,__pycache__,node_modules,bower_components
+ignore = D203
+max-complexity = 7
+max-line-length = 120
+putty-ignore =
+    **/__init__.py : +F401
+    app/__init__.py, app/main/__init__.py, app/status/__init__.py : +E402
+    app/main/views/*.py : +C901
```

Comment:
Because we perform so much logic in our views, and because our codebase effectively dictates that's where the logic needs to live, it means our views already have high levels of complexity.
I'm sort of OK with that if we keep the complexity down everywhere else. I think it'd be good to have low complexity everywhere but it didn't look viable for a big bang PR.